front-end/public/index.html

```diff
@@ -4,6 +4,9 @@
     <meta charset="utf-8">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.png" type="image/png">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css" integrity="sha384-gfdkjb5BdAXd+lj+gudLWI+BXq4IuLW5IT+brZEZsLFm++aCMlF1V92rMkPaX4PP" crossorigin="anonymous">
```

Comment:
Yeah, I know about using FontAwesome icons. I'he already tried using this, but it doesnt work because of using react scopes. I can't use it like this '<FontAwesome/>' (import does not visible here). And without scopes it's Error. So we discussed this with Yaroslav and left it just like this.